src/Core/Usecase/Tos/CreateTos.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * Add post to Tos Usecase
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Platform
+ * @copyright  2017 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+namespace Ushahidi\Core\Usecase\Tos;
+
+use Ushahidi\Core\Entity;
+use Ushahidi\Core\Entity\Tos;
+use Ushahidi\Core\Usecase\CreateUsecase;
+
+class CreateTos extends CreateUsecase
```

Comment:
You don't need to extend this if nothing changes, you can just use CreateUsecase as-is